packages/server/utils/analytics/analytics.ts

```diff
@@ -272,6 +273,13 @@ class Analytics {
       isNewUser,
       acceptAt
     })
+
+    this.track(inviterId, 'Sent Invite Accepted', {
+      teamId,
+      userId,
```

Comment:
-1: rename this to `inviteeId` for better clarity?

Reply:
Fixed!